Utils/pom.xml

```diff
@@ -84,20 +84,30 @@
                 <artifactId>maven-checkstyle-plugin</artifactId>
                 <version>2.17</version>
                 <dependencies>
-	                <dependency>
-	                    <groupId>com.microsoft.azuretools</groupId>
-	                    <artifactId>check-tools</artifactId>
-	                    <version>1.0.0</version>
-	                </dependency>
-	                <dependency>
-	                    <groupId>com.puppycrawl.tools</groupId>
-	                    <artifactId>checkstyle</artifactId>
-	                    <version>7.7</version>
-	                </dependency>
+                    <dependency>
+                        <groupId>com.microsoft.azuretools</groupId>
+                        <artifactId>check-tools</artifactId>
+                        <version>1.0.0</version>
+                    </dependency>
+                    <dependency>
+                        <groupId>com.puppycrawl.tools</groupId>
+                        <artifactId>checkstyle</artifactId>
+                        <version>7.7</version>
+                    </dependency>
```

Comment:
The same as above.

Reply:
Here the indentation is wrong, but I'll follow your suggestion to fix it in another PR.